### ISSUE TITLE
Publish binaries

### ISSUE THREAD

**Issue by petemounce** [steven-turner/sonar-auth-google#4]
Hi there - would it be possible to get you to publish build artifacts into github releases for easier distribution, please?

**Comment by steven-turner** [comment]
Yes, I'll set aside some time this weekend to bring this project back up to date. Thanks for the feedback.

**Comment by petemounce** [comment]
Thanks very much!
